lib/Dialog/Dialog.js

```diff
@@ -144,7 +154,9 @@ module.exports = kind(
 			]}
 		]},
 		{kind: Divider, classes: 'moon-dialog-divider'},
-		{name: 'message', kind: BodyText, classes: 'moon-dialog-content'},
+		{name: 'bodyComps', components: [
```

Comment:
Hi @rajyavardhanp ,
I think this will work without wrapping message with another div. We can just create bodyComponents on message not bodyComps.